packages/clerk-js/src/ui/components/SignUp/SignUpStart.tsx

```diff
@@ -39,10 +39,9 @@ function _SignUpStart(): JSX.Element {
   const { attributes } = userSettings;
   const { setActive } = useClerk();
   const ctx = useSignUpContext();
-  const options = useOptions();
   const isWithinSignInContext = !!React.useContext(SignInContext);
-  const { afterSignUpUrl, signInUrl, unsafeMetadata } = ctx;
-  const isCombinedFlow = !!(options.experimental?.combinedFlow && !!isWithinSignInContext);
+  const { afterSignUpUrl, signInUrl, unsafeMetadata, isCombinedFlow: _isCombinedFlow } = ctx;
+  const isCombinedFlow = !!(_isCombinedFlow && !!isWithinSignInContext);
```

Comment:
It might be better not to create a temporary `_isCombinedFlow` 
```suggestion
  const { afterSignUpUrl, signInUrl, unsafeMetadata } = ctx;
  const isCombinedFlow = !!(ctx.isCombinedFlow && !!isWithinSignInContext);
```